src/NzbDrone.Core/Localization/Core/en.json

```diff
@@ -2138,6 +2138,9 @@
   "UsenetDelayTime": "Usenet Delay: {usenetDelay}",
   "UsenetDisabled": "Usenet Disabled",
   "UserInvokedSearch": "User Invoked Search",
+  "UserRejectedExtensions": "Additional Rejected File Extensions",
+  "UserRejectedExtensionsHelpText": "Comma separated list of files extensions to fail (Fail Downloads also needs to be enabled per indexer)",
+  "UserRejectedExtensionsTextsExamples": "Examples: '.001, .dog' or '001,dog'",
```

Comment:
Should we treat `001` is an archive file as that's typically the use of them?

Reply:
> Should we treat `001` is an archive file as that's typically the use of them?

I think it would make sense to include the 001 in the list of archive extensions, but nothing more than that. want separate PR for that?
I'll update the helptext here